### DISCUSSION TITLE
Status: Offline, Version: Unknown

### DISCUSSION THREAD

**Discussion by Cirion75** [immich-app/immich#7145] [Q&A] [answered]
New installation, all photos & videos imported, and everything seems to work.
But why is the status like this?
![image](https://github.com/immich-app/immich/assets/1172168/28ed28ee-596b-4ca1-9877-5bb2de0942bf)

**Comment by bo0tzz** [comment, marked as answer]
You're probably using a reverse proxy that isn't passing through websocket connections properly.

**Comment by Cirion75** [comment]
Quick fixes are always the best :)
![image](https://github.com/immich-app/immich/assets/1172168/698c417a-e429-4bd4-bf00-afc0129240bc)
